src/store/lookup.ts

```diff
@@ -531,6 +544,59 @@ async function runCaptchaDeterrent(browser: Browser, store: Store, page: Page) {
   }
 }
 
+async function handleCaptchaAsync(page: Page, store: Store) {
+  // set up element queries
+  const imageElementQuery = {
+    requireVisible: true,
+    selector: store.labels.captchaHandler?.image || 'img',
+  };
+  const inputElementQuery = {
+    requireVisible: true,
+    selector: store.labels.captchaHandler?.input || 'input',
+  };
+  const submitElementQuery = {
+    requireVisible: true,
+    selector: store.labels.captchaHandler?.submit || 'button[type="submit"]',
+  };
+
+  // get image src for captcha
+  const imgElementSrc = await page.evaluate((selector: string) => {
+    const element = document.querySelector<HTMLImageElement>(selector);
+    return element?.src;
+  }, imageElementQuery.selector);
+
+  const response = await getCaptchaInputAsync(
+    imgElementSrc ||
+      `captcha detected on [${page.url()}] but unable to get captcha image url`
```

Comment:
It may be a good idea to figure out how to do this without sending another captcha download request, it may fail/get detected on some websites. 

I think it was possible via Canvas like this

```
const cnv = document.createElement('CANVAS');
const ctx = cnv.getContext('2d');
ctx.drawImage(img, 0, 0);
const base64ImgData = cnv.toDataURL();
```

Reply:
Looks like here are some solutions for puppeter: https://intoli.com/blog/saving-images/